en_US/changes/changes-ee-4.4.md

```diff
@@ -1,5 +1,27 @@
 # Releases
 
+## e4.4.21
+
+*Release Date: 2023-10-13*
+
+### Enhancements
+
+- Added support for Confluent data bridge.
+
+- Now the MQTT topic field in the Kafka consumer group supports templates with placeholders.
+
+  For example, if the key of the message consumed by Kafka is "a", and the configured MQTT topic is "topic/${key}", then when the message is forwarded, the MQTT topic will be replaced with "topic/a".
+
+- Now the "Message Republish" action supports adding MQTT properties and user properties.
+
+  The "Message Republish" action supports two new fields: "MQTT Properties" and "User Property Value", both of which are in the format of key-value pairs, and both the key and value support placeholders.
+
+### Bug fixes
+
+- Fixed the issue that the Kafka action cannot send numeric values as Kafka Headers.
+
+  Prior to this fix, when the "Kafka Headers value encode mode" is set to "NONE", if the "Kafka Headers" field contains a JSON object with numeric types (such as `{"a": 1, "b": "str"}`), the numeric values (`"a":1`) will be ignored and not sent to Kafka. After the fix, the numeric types in JSON will be converted to strings before being sent to Kafka.
```

Comment:
strings -> string?

Reply:
"numeric types be converted to strings. ", I am not sure if there's any grammar problems in it, I just used the `copilot`(AI) to generate the translation for me.